adapters/audienceNetwork/facebook.go

```diff
@@ -41,7 +42,7 @@ type facebookReqExt struct {
 	AuthID     string `json:"authentication_id"`
 }
 
-func (this *FacebookAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+func (a *FacebookAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
```

Comment:
Nitpick: I like the idea of changing the variable name here, I would suggest it to be `fa` to represent `FacebookAdapter`, but I'm fine with `a` as well.

Reply:
Good point. It really should be "adapter" instead of "FacebookAdapter" though. I'll make that change instead and keep the "a" variable. Cool?

Reply:
Yeah that works!